lib/msf/core/exploit/http/client.rb

```diff
@@ -334,7 +334,7 @@ def send_request_raw(opts={}, timeout = 20)
   # Passes `opts` through directly to {Rex::Proto::Http::Client#request_cgi}.
   #
   # @return (see Rex::Proto::Http::Client#send_recv))
-  def send_request_cgi(opts={}, timeout = 20)
+  def send_request_cgi(opts={}, timeout = 20, disconnect = true)
```

Comment:
As much as I hate tacking on new parameters, `disconnect` is more like `timeout` than `opts`. Maybe one day we'll fully realize keyword arguments.